Mathlib/Analysis/Normed/Ring/Seminorm.lean

```diff
@@ -388,3 +388,52 @@ lemma MulRingNorm_nat_le_nat {R : Type*} [Ring R] (n : ℕ) (f : MulRingNorm R)
       f (n + 1) ≤ f (n) + f 1 := f.add_le' ↑n 1
       _ = f (n) + 1 := by rw [map_one]
       _ ≤ n + 1 := add_le_add_right hn 1
+
+noncomputable section
+variable {R : Type*} [Ring R]
+variable {f g : MulRingNorm ℚ}
+
+/-- A multiplicative norm composed with the absolute value on integers equals the norm itself. -/
+lemma f_of_abs_eq_f (x : ℤ) (f : MulRingNorm ℚ) : f (Int.natAbs x) = f x := by
+  obtain ⟨n,rfl|rfl⟩ := Int.eq_nat_or_neg x
+  · simp only [Int.natAbs_ofNat, Int.cast_natCast]
+  · simp only [Int.natAbs_neg, Int.natAbs_ofNat, Int.cast_neg, Int.cast_natCast, map_neg_eq_map]
+
+/-- Values of a multiplicative norm of the rationals are determined by the values on the
+integers. -/
+lemma NormRat_eq_on_Int_iff_eq_on_Nat : (∀ n : ℕ , f n = g n) ↔ (∀ n : ℤ , f n = g n) := by
```

Comment:
I am not sure about the name of this and the next two lemmas.

Reply:
Any suggestion is welcome!

Reply:
Maybe the could be moved to the `Rat.MulRingNorm` namespace. So this would become `Rat.MulRingNorm.eq_on_Int_iff_eq_on_Nat`, and similarly for the other lemmas.

Reply:
Uhmm sorry for the silly question, but how do I do it? Do I move the theorems to that file or is there some command to do it in this file? Sorry again 😅

Reply:
I think it works like a section: you open it with `namespace Rat.MulRingNorm` and close it with `end Rat.MulRingNorm`.

Reply:
I am fixing it.

Reply:
(this is very easy, since it is in a file you don't care about)

Reply:
Done (the bot is not super fast to remove the label, but it will do it). Don't forget to get a new cache after pulling.

Reply:
I think `NormRat` should not be part of these three lemma names (since they are already in the `Rat.MulRingNorm` space). So the names should be `eq_on_Int_iff_eq_on_Nat`, `eq_iff_eq_on_Nat` and `equiv_iff_equiv_on_Nat`.
Up to this change, the PR looks good to me, but I think it should be approved by someone not involved with the project.